### PR TITLE
Added skeleton HTML5 to the app

### PR DESCRIPTION
**Pull request by SHarpreet89**
HTML5 Starter code has been added to index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (cbfcuh) [inline index.html:6]: can you change the title to PageFlix?

**Review by cbfcuh — CHANGES_REQUESTED** [review]
Left a comment about the title

File: index.html (hunk 1 above)
Comment (SHarpreet89) [reply index.html]: Done!

**Review by cbfcuh — APPROVED** [review]
LGTM 👍

**Comment by SHarpreet89** [comment]
This addreses #11

**Pull request merged**